### PR TITLE
Composer 2.4.0 => 2.4.1

### PR DESCRIPTION
**Pull request by uberhacker**

### CODE REVIEW

File: packages/composer.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@ def self.preinstall
 
   def self.install
     downloader "https://github.com/composer/composer/releases/download/#{version}/composer.phar",
```
Comment (supechicken) [inline packages/composer.rb:27]: Use `source_url`?

Comment (uberhacker) [reply packages/composer.rb:27]: Does that work with `.phar` files?

Comment (supechicken) [reply packages/composer.rb:27]: Yes, any non-archive files should work

Comment (uberhacker) [reply packages/composer.rb]: Done.  I also abstracted the path from `~/.bashrc`.

**Review by satmandu — APPROVED** [review]

**Pull request merged**